File: messageDotNet/grpc/MethodData/Team.cs
using System;
using UnityEngine;

/// <summary>
/// call friend service, get team data for user id  
/// </summary>
[Serializable]
public class TeamDataInput
{
    public long UserId;
}
[Serializable]
public class TeamDataOutput
{
    public bool Success;
    public string ErrMsg;
    public bool exist;
    public Team TeamData;
}

File: messageGo/grpc/methodData/team.go
package methodData

import (
	base_data "game-message-core/grpc/baseData"
)

type TeamDataInput struct {
	UserId int64 `json:"userId"`
}

func (p *TeamDataInput) Clear() {
	p.UserId = 0
}

type TeamDataOutput struct {
	Success  bool           `json:"success"`
	ErrMsg   string         `json:"errMsg"`
	Exist    bool           `json:"exist"`
	TeamData base_data.Team `json:"teamData"`
}

func (p *TeamDataOutput) Clear() {
	p.Success = false
	p.ErrMsg = ""
	p.Exist = false
	p.TeamData.Clear()
}
